### DISCUSSION TITLE
Default install prefix is not /usr/local

### DISCUSSION THREAD

**Discussion by just-max** [polybar/polybar#2949] [Build Support] [answered]
The [wiki on compiling](https://github.com/polybar/polybar/wiki/Compiling) suggests:

```bash
# Optional. This will install the polybar executable in /usr/local/bin
```

Is this a misconfiguration on my part, or does this typically install to `/usr/bin`? I would suggest adding a note:

```diff
mkdir build
cd build
- cmake ..
+ cmake .. # or, to install in /usr/local: cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr/local/ ..
make -j$(nproc)
- # Optional. This will install the polybar executable in /usr/local/bin
+ # Optional. This will install the polybar executable in /usr/bin (or /usr/local/bin, if configured above)
sudo make install
```

Not sure how to add pull requests to Github wikis, else I'd do that.

**Comment by patrick96** [comment, marked as answer]
The wiki reflects the state of the latest stable release (sometimes unreleased features are also mentioned, but they are explicitly tagged as unreleased).

Version 3.6.3 installs to `/usr/local/` by default. The change to `/usr/` was made just recently and the wiki will be updated once it is part of a release.

**Reply by just-max** [reply]
That makes sense, thanks!
